openam-oauth2/src/main/java/org/forgerock/oauth2/core/AuthorizationCodeRequestValidatorImpl.java

```diff
@@ -12,19 +12,21 @@
  * information: "Portions copyright [year] [name of copyright owner]".
  *
  * Copyright 2014 ForgeRock AS.
+ * Portions Copyright 2023 Wren Security.
  */
 
 package org.forgerock.oauth2.core;
 
+import static org.forgerock.openam.oauth2.OAuth2Constants.Params.CODE;
+import static org.forgerock.openam.oauth2.OAuth2Constants.Params.REDIRECT_URI;
+
+import javax.inject.Inject;
+
```

Comment:
Separating imports into groups violates our style guide - https://github.com/WrenSecurity/wrensec-docs/wiki/Java-Style-Guide#333-ordering-and-spacing .

Reply:
Yes, you're right. I had an invalid import order configuration in in my IDE. It should be fixed now.